src/Platform/Microsoft.Testing.Platform/OutputDevice/Console/TestProgressState.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.OutputDevice.Console;
+
+internal class TestProgressState
+{
+    public TestProgressState(int passed, int failed, int skipped, string assemblyName, string? targetFramework, string? architecture, StopwatchAbstraction stopwatch, string? detail)
```

Comment:
Any reason to have tfm and architecture nullable and to throw when null? It's better to mark as non-nullable.

Reply:
No it is autogenerated code oversight. Fixed.